src/Viewer.tsx

```diff
@@ -107,8 +97,8 @@ function Viewer(): ReactElement {
   const workerPool = getSharedWorkerPool();
   const arrayLoader = useConstructor(() => new UrlArrayLoader(workerPool)).current;
 
-  // TODO: Refactor dataset dropdowns, color ramp controls, and time controls into separate
-  // components to greatly reduce the state required for this component.
+  // TODO: Break down into separate components to greatly reduce the state
+  // required for this component.
```

Comment:
There are still a few components (like the color ramp dropdown) that will also be moved in future PRs.